Index: x11-common/etc/X11/Xserver
===================================================================
--- x11-common.orig/etc/X11/Xserver
+++ x11-common/etc/X11/Xserver
@@ -23,7 +23,7 @@ module_id() {
     # grep "Module ID" /proc/hal/assets | sed "s/.*://"
     ## used to read from /proc/hal/model, but that is removed in 2.6
     # echo ' iPAQ' `cat /proc/hal/model`
-    awk 'BEGIN { FS=": " } /Hardware/ { print $2 } ' </proc/cpuinfo
+    awk 'BEGIN { FS=": " } /platform/ { print $2 } ' </proc/cpuinfo
 }

 ARGS=" -br -pn"
